apps/dapp/src/components/Pages/Core/DappPages/SpiceBazaar/Analytics/Chart/componentsOfChart/EmissionAllocationStaked.tsx

```diff
@@ -0,0 +1,85 @@
+import { subDays } from 'date-fns';
+import LineChart from '../../components/LineChart';
+import { formatNumberAbbreviated } from 'utils/formatter';
+import { useTheme } from 'styled-components';
+import { tickFormatter } from '../Chart';
+import type { Metric } from '../Chart';
+import Loader from 'components/Loader/Loader';
+
+const pricesLast7Days: Metric[] = [
+  {
+    timestamp: subDays(new Date(), 6).getTime(),
+    epoch1: 1.13,
+    epoch2: 1.14,
+    epoch3: 1.15,
+  },
+  {
+    timestamp: subDays(new Date(), 5).getTime(),
+    epoch1: 1.14,
+    epoch2: 1.15,
+    epoch3: 1.16,
+  },
+  {
+    timestamp: subDays(new Date(), 4).getTime(),
+    epoch1: 1.13,
+    epoch2: 1.14,
+    epoch3: 1.15,
+  },
+  {
+    timestamp: subDays(new Date(), 3).getTime(),
+    epoch1: 1.14,
+    epoch2: 1.15,
+    epoch3: 1.16,
+  },
+  {
+    timestamp: subDays(new Date(), 2).getTime(),
+    epoch1: 1.16,
+    epoch2: 1.17,
+    epoch3: 1.18,
+  },
+  {
+    timestamp: subDays(new Date(), 1).getTime(),
+    epoch1: 1.15,
+    epoch2: 1.16,
+    epoch3: 1.17,
+  },
+  { timestamp: new Date().getTime(), epoch1: 1.17, epoch2: 1.18, epoch3: 1.19 },
+];
+
+const EmissionAllocationStaked = () => {
+  const metrics = pricesLast7Days;
+  const theme = useTheme();
+
+  const formatWithCommas = (num: number) => {
+    return new Intl.NumberFormat('en-US').format(num);
+  };
+
+  if (!metrics.length) return <Loader />;
+  return (
+    <LineChart
+      chartData={metrics.reverse()}
+      xDataKey="timestamp"
+      lines={[
+        { series: 'epoch1', color: theme.palette.brandLight },
+        { series: 'epoch2', color: theme.palette.brandDark },
+        { series: 'epoch3', color: '#D0BE75' },
+      ]}
+      xTickFormatter={tickFormatter}
+      yTickFormatter={(val) =>
+        `$${formatNumberAbbreviated(val).number.toFixed(2)}\u00A0M`
+      }
+      tooltipLabelFormatter={tickFormatter}
+      tooltipValuesFormatter={(value) => [`$ ${value.toFixed(2)} M`, 'Value']}
+      legendFormatter={(value) =>
```

Comment:
probably another way to do this :

```
const epoch = value.match(/\d+/)[0]
return `EPOCH ${epoch}`
```

Reply:
Everything under `Analytics` is hidden and not in use today. I'll make a note of this for whenever we go to ship the Analytics page.